### ISSUE TITLE
Unable to use Treatment Plan template in Patient Encounter 

### ISSUE THREAD

**Issue by yadavyk** [earthians/marley#150]
### Information about bug

Issue : When trying to use Treatment Plan template from Patient Encounter, on hitting "Get Applicable Treatment Plans" button in Encounter Impression section, no preconfigured items, drugs, clinical procedures or lab tests are fetched. It also creates a duplicate Patient Encounter record 

Snippet : 

![Treatment Plan Template](https://user-images.githubusercontent.com/32797974/155666584-b3cb6789-7e3d-466c-ac11-1466c793b4e4.gif)

### Module

other

### Version

V13.21.0

**Comment by muhammadmp** [comment]
@pjamsheer

**Comment by praveen7488** [comment]
I agree. When I tried using the Treatment Plan Template from Patient Encounter, on hitting the "Get Applicable Treatment Plans" button in the Encounter Impression section, no preconfigured items, drugs, clinical procedures, or lab tests are fetched.

Not sure how to solve this though. 

I tried in ERPNext: v13.36.2 (version-13); Frappe Framework: v13.36.3 (version-13).

**Comment by mujeerhashmi** [comment]
> I agree. When I tried using the Treatment Plan Template from Patient Encounter, on hitting the "Get Applicable Treatment Plans" button in the Encounter Impression section, no preconfigured items, drugs, clinical procedures, or lab tests are fetched.
> 
> 
> 
> Not sure how to solve this though. 
> 
> 
> 
> I tried in ERPNext: v13.36.2 (version-13); Frappe Framework: v13.36.3 (version-13).

The patient encounter should be first saved to be able to use treatment plans.

**Comment by Saurav0786** [comment]
Even after fetching the data from template in Draft mode, its still did not show the count of session in the therapy type.

**Comment by akashkrishna619** [comment]
Thanks for reporting, we will fix this.

**Comment by praveen7488** [comment]
@akashkrishna619 We are continuing to face the issue in Treatment Plan Template. Can you please check

Description of the Issue
Number of sessions field is not getting fetched from a Treatment Plan Template when fetching from Patient Encounter doc type.

Treatment Plan Template Screenshot
![image](https://user-images.githubusercontent.com/117389387/234241891-63e638a9-74c5-46cb-9206-d09d5f2ed1ac.png)

Patient Encounter Screenshot post fetching the Treatment Plan Template. The No of Sessions/Quantity field is not getting fetched.
![image](https://user-images.githubusercontent.com/117389387/234242035-f604c064-98d2-403b-8f74-692b6a9c60c0.png)

Version Details:
ERPNext: v14.17.2 (version-14)
flexitude_app: v0.0.1 (development)
Frappe Framework: v14.26.2 (version-14)
Healthcare: v15.0.0-dev (develop)
Frappe HR: v14.0.3 (version-14)
India Compliance: v14.1.2 (version-14)
Frappe LMS: v0.0.1 (main)
Payments: v0.0.1 (develop)

**Comment by akashkrishna619** [comment]
@praveen7488 it is working on develop branch. If you have pulled latest version of Healthcare, please do a bench restart as well.
